awx/ui/src/screens/Job/JobOutput/useJobEvents.js

```diff
@@ -83,6 +97,7 @@ export function jobEventsReducer(callbacks, isFlatMode, enqueueAction) {
       ...origState,
       events: { ...origState.events },
       tree: [...origState.tree],
+      isAllCollapsed: origState.isAllCollapsed,
```

Comment:
probably not necessary since `...origState` is already spread into the object a couple lines above